tests/system/marathon_common_tests.py

```diff
@@ -1133,3 +1133,33 @@ def http_output_check():
         assert 'Relay from {}'.format(relay_app["id"]) in output
 
     http_output_check()
+
+
+@shakedown.dcos_1_11
+def test_ipv6_healthcheck():
+    """ There is new feature in DC/OS 1.11 that allows containers running on IPv6 network to be healthchecked from
+        Marathon. This tests verifies executing such healthcheck.
+    """
+    agents = shakedown.get_agents()
+    # to test ipv6 healthcheck we need ipv6 network before launching the container
+    # the container launch then uses this network in the app definition json
+    network_cmd = "sudo docker network create --driver=bridge --ipv6 --subnet=fd01::/64 mesos-docker-ipv6-test"
```

Comment:
This (and removing the network afterwards) should be done as a fixture. Otherwise the cleanup might not be executed.

Reply:
good point, done